src/components/CommentList/Comment/Comment.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Comment.scss';
+
+export const Comment = ({ oneComment }) => (
+  <div className="commentaries">
```

Comment:
```suggestion
  <div className="commentary">
```
this is single comment